services/src/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/switchmanager/ISwitchManager.java

```diff
@@ -191,6 +192,14 @@ long installOneSwitchFlow(final DatapathId dpid, final String flowId, final Long
                                                       int outputVlanId, final OutputVlanType outputVlanType,
                                                       final long meterId) throws SwitchOperationException;
 
+    /**
+     * Returns list of default flows that must be installed on a switch.
+     *
+     * @param dpid switch id.
+     * @return list of default flows.
+     */
+    List<OFFlowMod> getDefaultFlows(DatapathId dpid) throws SwitchOperationException;
```

Comment:
I think it a bad naming because `getDefaultFlows` should returns default flows which already installed on switch

Reply:
what if `getExpectedDefaultFlows`